### PR TITLE
[PORT] Cryopods and Ghost Spawner Announcements!

### PR DESCRIPTION
**Pull request by RimiNosha**
# Requires #216!

## About The Pull Request

Ports cryopods in their entirety from SR. I did fix a weird logic issue, as well as a funny moment where I forgot to patch the AI cryo verb to use the right announcing system, so here we are.

Ripping out ghost spawner announcements would be too much work, plus I want them anyways, so this works for me.

I removed the prison pods, though, because wall-mounted cryo is not something I'm even remotely interested in.

Credit to Azarak for original code, Gandalf and myself for a few notable improvements, and many folk over at skyrat for the bugfixes.

## How Does This Help ***Gameplay***?

Seamless way to leave the round!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary> 

image
Oh look, it cryos.
image
Cryo'd with item
image
I get da item

</details>

## Changelog

:cl:
add: Cryopods! Leave the round in style!
/:cl:

### CODE REVIEW

File: code/modules/cryosleep/readme.md
[the remark this answers is not in the document]
Comment (RimiNosha) [reply code/modules/cryosleep/readme.md:1]: Uhhhhh, what and how? Will fix.

File: code/modules/cryosleep/admin.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	var/datum/effect_system/spark_spread/quantum/sparks = new
+	sparks.set_up(10, 1, loc)
+	sparks.attach(loc)
+	sparks.start()
+
+	//make a paper if need
+	if(with_paper)
+		var/obj/item/paper/cryo_paper = new /obj/item/paper(loc)
+		cryo_paper.name = "Notification - [name]"
+		cryo_paper.add_raw_text("Our sincerest apologies, [name][job ? ", [job]," : ""] had to be sent back in Cryogenic Storage for reasons that cannot be elaborated on at the moment.<br><br>Sincerely,<br><i>Nanotrasen Anti-Sudden Sleep Disorder Agency</i>")
```
[the remark this answers is not in the document]
Comment (RimiNosha) [reply code/modules/cryosleep/admin.dm:14]: Yea, now that you've pointed it out, this is kinda crappy.

**Comment by QuacksQ** [comment]
Would it be possible to add an auto cryo option to this? For people that have been SSD so they get automatically removed from the round of they have been afk for too long

**Comment by RimiNosha** [comment]
Automatic cryoing of folk is not something I'm interested in. Other players can do it after a certain amount of time has been elapsed.

**Comment by RimiNosha** [comment]
TM only for now, causes harddels on cryo, and if I fullmerge this, it'll never get fixed.

**Comment by RimiNosha** [comment]
Something to do with DNA. Look closer when I'm not falling asleep on the spot.

**Comment by nevimer** [comment]
what could go wrong?

**Pull request merged**